pulpcore/cli/common/generic.py

```diff
@@ -1448,3 +1460,58 @@ def content_group() -> None:
             content_group.add_command(command)
 
     return content_group
+
+
+def _version_list_callback(
```

Comment:
Why is this function hiding away at the end of the file far away from the other fellow callbacks? Also don't make an export start with `_`.